### DISCUSSION TITLE
2022/10/23/%E6%97%A5%E8%AE%B0-2022-10-23/

### DISCUSSION THREAD

**Discussion by giscus** [StarCried/StarCried.github.io#6] [Announcements]
# 2022/10/23/%E6%97%A5%E8%AE%B0-2022-10-23/

StarCried 的狼窝，记录算法竞赛知识（OI, ACM）以及学习、工作、生活中的点点滴滴。 - StarCried - StarCried 的狼窝

https://starcried.github.io/2022/10/23/%E6%97%A5%E8%AE%B0-2022-10-23/

**Comment by ttyaaron** [comment]
（这里也有一个小时候喜欢在晚上默默流泪的人。
